src/theme/selectors.js

```diff
@@ -0,0 +1,16 @@
+import { get, isObject } from 'lodash';
+
+export const selector = (propsOrSelector, prevArgs = []) => {
+  if (isObject(propsOrSelector)) {
+    return get(propsOrSelector, prevArgs);
+  }
+
+  return value => selector(value, [...prevArgs, propsOrSelector]);
+};
```

Comment:
From what I can see with how this function is currently used, we could simply have:

```
export const selector = key => (
  obj => get(obj, key)
);
```

Or is there another use case you have in mind that is currently not expressed here?

Reply:
Ah yes, I see.. Cool, in that case, I'd just copy your comment in the code to make it clear what the use case is and then it'd be ok to merge!